ports/nrf/fatfs_port.c

```diff
@@ -26,8 +26,18 @@
 
 #include "py/runtime.h"
 #include "lib/oofatfs/ff.h"
+#include "lib/timeutils/timeutils.h"
+
+extern void common_hal_rtc_get_time(timeutils_struct_time_t *tm);
+extern void struct_time_to_tm(mp_obj_t t, timeutils_struct_time_t *tm);
```

Comment:
`#include` `shared-bindings/rtc/RTC.h` and `shared-bindings/time__init__.h` to get these definitions.

Reply:
@dhalbert Thanks, did that. 
-ttmetro